packages/discovery-provider/ddl/migrations/0100_shorten_challenge_ids.sql

```diff
@@ -0,0 +1,246 @@
+/** 
```

Comment:
can we maybe get a centralized readme or something that maps all of these to their meanings? so we don't have to dig through migration files in the future.

i.e.

`p` -> `profile-completion`

also, there's a few `0100` migrations already, maybe we can bump this up

Reply:
The SDK has named enums for these! But yeah would be nice if we had constants in the python code

Reply:
yea noticed the SDK enum so removed that part of the comment 😅